src/components/RelatedArticle/RelatedArticle.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+// images
+import scene from '../../assets/finepic.png';
```

Comment:
Nice work Gift. The feedback I got from the TTL on image naming is that I should properly name my image imports. e.g scene should be sceneImage | sceneLogo | sceneIcon depending on what it represents. I think you should do the same.

Reply:
Thanks I'll go with this format.